tailwind.config.cjs

```diff
@@ -74,6 +74,7 @@ module.exports = {
         600: tailwindColors.violet[600],
         700: tailwindColors.violet[700],
       },
+      'github-star-icon': '#e3b341'
```

Comment:
it should not go to the tailwind config

Reply:
Do I hardcode the value ?

Reply:
you add it into the color-registry class

(it means that an extension would be able to override the color value and we can have different values for light and dark mode)

Reply:
side question: we can't use one value of the amber palette color ?

Reply:
Hummm I used the color from the GitHub website